beacon_node/src/cli.rs

```diff
@@ -759,9 +759,23 @@ pub fn cli_app() -> Command {
             Arg::new("slots-per-restore-point")
                 .long("slots-per-restore-point")
                 .value_name("SLOT_COUNT")
-                .help("Specifies how often a freezer DB restore point should be stored. \
-                       Cannot be changed after initialization. \
-                       [default: 8192 (mainnet) or 64 (minimal)]")
+                .help("DEPRECATED. This flag has no effect.")
```

Comment:
There are a few more references of this args that needs to be removed, e.g.:

https://github.com/sigp/lighthouse/blob/3070cb7c3982f8321bbeb33413009f8f928bdf8a/book/src/advanced_database.md#L16

https://github.com/sigp/lighthouse/blob/7b1a0d4bba4d6ad8c85273745e273fd8cdf74cc6/watch/README.md#L42

Reply:
updated the docs in this commit: https://github.com/sigp/lighthouse/pull/5978/commits/e87a618a29d1cce7060dee8607e5eb3b3c29eb13